src/app/community/_components/OrderListModal.tsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames/bind';
+
+import { Button } from '@/components';
+import type { PostCardDetailModalCustomKeyboardType } from '@/types/CommunityTypes';
+import Image from 'next/image';
+import { keydeukImg } from '@/public/index';
+
+import styles from './OrderListModal.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface OrderListModalProps {
+  orderList: PostCardDetailModalCustomKeyboardType[];
+  onOpenReviewModal: () => void;
+  onSelectProduct: (i: number) => void;
+  selectedOrder: PostCardDetailModalCustomKeyboardType | null;
+}
+
+export default function OrderListModal({
+  orderList,
+  onOpenReviewModal,
+  onSelectProduct,
+  selectedOrder,
+}: OrderListModalProps) {
+  const handleClickWriteButton = () => {
+    onOpenReviewModal();
+  };
+
+  return (
+    <div className={cn('container')}>
+      <h1 className={cn('title')}>작성할 후기 제품을 선택해주세요.</h1>
+      <div className={cn('keyboard-list-wrapper')}>
+        {orderList.map((order, i) => (
+          <div
+            className={cn('keyboard-list', `${order === selectedOrder && 'selected-list'}`)}
```

Comment:
객체로 보내줄 수 있을 것 같습니다~!
```suggestion
            className={cn('keyboard-list', {'selected-list' : order === selectedOrder})}
```